tests/source_data_validation/test_source_schemas.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+from jsonschema import validate
+
+from tests.source_data_validation import (
+    CLUSTER_MAPPING,
+    CLUSTER_MAPPING_SCHEMA,
+    CONTAINER_LOG_REQUESTS,
+    CONTAINER_LOG_SCHEMA,
+    GATHERING_RULE_SCHEMA,
+    GATHERING_RULES,
+    REMOTE_CONFIGURATIONS,
+    REMOTE_CONFIGURATIONS_V2_TEMPLATE_SCHEMA,
+)
+
+
+def test_cluster_mapping_schema():
+    schema = json.loads(CLUSTER_MAPPING_SCHEMA.read_text())
+    mapping = json.loads(CLUSTER_MAPPING.read_text())
+    validate(mapping, schema)
+
+
+def test_container_log_schema():
+    schema = json.loads(CONTAINER_LOG_SCHEMA.read_text())
+    for log_file in CONTAINER_LOG_REQUESTS.rglob("*.json"):
```

Comment:
Can we parametrize the test rather than iterating over files inside the test?

Reply:
Test has been parametrized